ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll

```diff
@@ -48,12 +48,15 @@ module UnsafeDeserialization {
   }
 
   /**
-   * An argument in a call to `YAML.load`, considered a sink for unsafe
-   * deserialization.
+   * An argument in a call to `YAML.load` or `YAML.load_file`, considered a sink
+   * for unsafe deserialization. As the `YAML` module is an alias of `Psych` in
```

Comment:
Looks like you forgot to finish the sentence here.

Reply:
Oops, thanks